Dockerfile-dev

```diff
@@ -11,7 +11,7 @@ RUN useradd -d /home/zulip -m zulip && echo 'zulip ALL=(ALL) NOPASSWD:ALL' >> /e
 USER zulip
 
 RUN ln -nsf /srv/zulip ~/zulip
-
+RUN echo 'export LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8"' >> ~zulip/.bashrc
```

Comment:
It is more succinct with `tee` which can redirect to multiple files.

Reply:
I'm not sure `tee` is guaranteed to be installed in the container at this point.